docs/application/native/guides/multimedia/media-controller.md

```diff
@@ -196,11 +336,242 @@ To process the received command on the server side:
+   {
+       dlog_print(DLOG_DEBUG, LOG_TAG, "Server Name: %s, Request Id: %s, Result Code: %d\n", server_name, request_id, result_code);
+   }
+   ```
+
+2. Register the callback:
+
+   ```
+   ret = mc_client_set_cmd_reply_received_cb(g_client_h, cmd_reply_received_cb, NULL);
+   ```
+
+> **Note**
+>
+> This feature supports Tizen 4.0 and higher for Mobile.
+
+<a name="send_event"></a>
+## Sending and Processing A Custom Event
+
+To send a custom event to the client from the server side:
+
+1. Create the media controller server handle using the `mc_server_create()`:
+
+   ```
+   ret = mc_server_create(&g_server_h);
+   ```
+
+2. Retrieve the client name using the `mc_server_foreach_client()`:
+
+   ```
+   bool
+   activated_client_cb(const char *client_name, void *user_data)
+   {
+     GList **client_list = (GList **)user_data;
+
+     if (!client_list || !client_name) return FALSE;
+     *client_list = g_list_append(*client_list, g_strdup(client_name));
+
+     return TRUE;
+   }
+
+   GList *client_list = NULL;
+
+   ret = mc_server_foreach_client(g_mc_server, activated_client_cb, &client_list);
+   ```
+
+3. Send the event to the client using the corresponding `mc_server_send_custom_event()`. Use the client name retrieved in the previous step to identify the client.
+For example, to send a your own event to the client, use the `mc_server_send_custom_event()` with the event in the third parameter:
+
+   ```
+   char *request_id = NULL;
+
+   ret = mc_server_send_custom_event(g_server_h, client_name, "evnet1", NULL, &request_id);
+   ```
+
+4. Destroy the media controller server handle using the `mc_server_destroy()`, when media controller server handle is no longer needed:
+
+   ```
+   mc_server_destroy(g_server_h);
+   ```
+
+To process the received event on the client side:
+
+1. Create the media controller client handle using the `mc_client_create()`:
+
+   ```
+   ret = mc_client_create(&g_client_h);
+   ```
+
+2. Define the callback that is invoked when the client receives the event.
+
+   For example, to define a callback for a custom event:
+
+   ```
+   void
+   event_received_cb(const char* server_name, const char *request_id, const char *event, bundle *data, void *user_data)
+   {
+       dlog_print(DLOG_DEBUG, LOG_TAG, "Server Name: %s, Request id: %s, Event: %s\n", server_name, request_id, event);
+   }
+   ```
+
+3. Register the callback:
+
+   - To register a callback for a custom events, use the `mc_client_set_custom_event_received_cb()`.
+
+      For example, to register a callback for a custom event:
+
+      ```
+      ret = mc_client_set_custom_event_received_cb(g_client_h, event_received_cb, NULL);
+      ```
+
+4. Destroy the media controller client handle using the `mc_client_destroy()`, when media controller client handle is no longer needed:
+
+   ```
+   mc_client_destroy(g_client_h);
+   ```
+
+
+<a name="send_event_reply"></a>
```

Comment:
As mentioned earlier, please avoid using HTML code. Use only MD specific code.

Reply:
The sentense is not header, so I changed it to header and The HTML tag changed to head anchor for MD.

Reply:
@rishu-samsung & @annie-samsung 
rishu-samsung misunderstood. I didn't tell you delete all HTML tags.
When markdown is rendered to HTML, the tool generates <a name=""></a> tag for a heading of markdown automatically. So...I guided SRI-B that you didn't need to add <a></a> tag for headings.

This is not a heading, and the writer wants to provide a link here.
Please don't give a wrong guide again.

Reply:
Hi @journey2w ,

Thanks for clarifying the same, I understand now that for non-heading hyperlinks we can use HTML tags, if required. Going forward I will keep this as note for my review process. Thanks again.